bin/database.py

```diff
@@ -387,36 +431,12 @@ def upgrade_schema():
     """
 
     db_version = get_db_version()
-    try:
-        if db_version < 1:
-            upgrade_to_1()
-        if db_version < 2:
-            upgrade_to_2()
-        if db_version < 3:
-            upgrade_to_3()
-        if db_version < 4:
-            upgrade_to_4()
-        if db_version < 5:
-            upgrade_to_5()
-        if db_version < 6:
-            upgrade_to_6()
-        if db_version < 7:
-            upgrade_to_7()
-        if db_version < 8:
-            upgrade_to_8()
-        if db_version < 9:
-            upgrade_to_9()
-        if db_version < 10:
-            upgrade_to_10()
-        if db_version < 11:
-            upgrade_to_11()
-        if db_version < 12:
-            upgrade_to_12()
-        if db_version < 13:
-            upgrade_to_13()
-        if db_version < 14:
-            upgrade_to_14()
 
+    try:
+        while db_version < CURRENT_DATABASE_VERSION:
+            db_version += 1
+            upgrade_script = 'upgrade_to_'+str(db_version)
+            globals().get(upgrade_script)()
```

Comment:
`globals()` seems to be the way to go for functions in the current module. Using the `globals()[upgrade_script]()` notation will produce a slightly more intuitive `KeyError` rather than a `TypeError: 'NoneType' object is not callable`. Might we worth catching either one of those explicitly and printing an appropriate message in order to expedite future debugging.